simplic-session/Simplic.Session.Service/SessionService.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Simplic.Session.Service
+{
+    /// <summary>
+    /// Session service holds a variable to the current session
+    /// </summary>
+    public class SessionService : ISessionService
+    {                
+        /// <summary>
+        /// Gets or sets the current session
+        /// </summary>
+        public Session CurrentSession
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        public SessionService()
+        {
+            CurrentSession = new Session();
```

Comment:
Shouldn't this be null if nothing is set?

Reply:
null or empty ?

Reply:
I would prefer null, than the check is simpler.